website/src/word-panel.css.tsx

```diff
@@ -0,0 +1,32 @@
+import { style } from "@vanilla-extract/css"
+import sprinkles from "./sprinkles.css"
+
+export const wholePanel = style({
+    position: "sticky",
```

Comment:
I never knew about this value for `position`! Looks like it has [pretty good browser support as long as you don't use it on table elements](https://caniuse.com/?search=sticky), but only if we don't do IE support. Looks like there's [a polyfill](https://github.com/wilddeer/stickyfill) if we do need to support IE. Great find!